python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -1677,4 +1677,102 @@
+      <Description>Time at which the data was read (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>matterConcentration1micron</EFDB_Name>
+      <Description>Particulate matter concentration for particles with diameter 1.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>ug/m3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>matterConcentration2point5micron</EFDB_Name>
+      <Description>Particulate matter concentration for particles with diameter 2.5 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>ug/m3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>matterConcentration4micron</EFDB_Name>
+      <Description>Particulate matter concentration for particles with diameter 4.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>ug/m3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>matterConcentration10micron</EFDB_Name>
+      <Description>Particulate matter concentration for particles with diameter 10.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>ug/m3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberConcentrationPoint5micron</EFDB_Name>
+      <Description>Number concentration of particles with diameter 0.5 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>count/cm3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberConcentration1micron</EFDB_Name>
+      <Description>Number concentration of particles with diameter 1.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>count/cm3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberConcentration2point5micron</EFDB_Name>
+      <Description>Number concentration of particles with diameter 2.5 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>count/cm3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberConcentration4micron</EFDB_Name>
+      <Description>Number concentration of particles with diameter 4.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>count/cm3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberConcentration10micron</EFDB_Name>
+      <Description>Number concentration of particles with diameter 10.0 um.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>count/cm3</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>typicalParticleSize</EFDB_Name>
+      <Description>Typical particle size.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>um</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>location</EFDB_Name>
+      <Description>The location of the sensor.</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>20</IDL_Size>
```

Comment:
IDL_Size is deprecated and is ignored. You can just remove it. Saying an attribute is a string with `<Count>1</Count>`, is enough to say this is a size-less string, which I think is wha you want.